packages/Popover/src/popover.scss

```diff
@@ -41,6 +42,7 @@
         --ids-popover-title-color: var(--hop-neutral-text);
         --ids-popover-title-font-weight: var(--hop-body-sm-semibold-font-weight);
         --ids-popover-title-padding: var(--hop-space-inset-xl);
+        --ids-popover-close-offset: 1rem;
```

Comment:
couldn't you use --hop-space-inline-md or something for this?

Reply:
Yep totally will fix